src/Patient/Information/index.js

```diff
@@ -2,41 +2,50 @@ import { Box, ButtonBase, Grid, Typography } from '@material-ui/core';
 import React from 'react';
 import { Route, Switch, Link } from 'react-router-dom';
 import { makeStyles } from '@material-ui/core/styles';
+import TopPageLabel from '../../Components/Shared/TopPageLabel';
+import { PageLabel } from '../../Components/Shared/PageLabel';
 
 const useStyles = makeStyles({
-    grid:{
+    grid: {
         display: "grid",
         gridTemplateColumns: "1fr 1fr",
         gridGap: "16px"
     },
-    box:{
+    box: {
         border: "solid 1px lightgray",
         borderRadius: "4px",
         display: "flex",
         aspectRatio: "1",
         alignItems: "center",
         justifyContent: "center",
-        "& *":{
+        "& *": {
             textAlign: "center"
         }
     }
 })
 
+const InfoRoute = (props) => {
+
+    const { title, children } = props;
+
+    return <Route {...props}>
+        <Box width="100%" bgcolor="white" zIndex={10} position="fixed" top={0}>
+            <PageLabel title={title} />
+        </Box>
+        {children}
+    </Route>
+}
+
```

Comment:
I like how you are still spreading out the props obj onto the Route component even though you've destructured it above with { title, children }. I always forget that I still have access to the og props after I destructure. This is a good reminder that certain components can pick and choose which of all props are most important to implement (btw, I am writing much of these comments out to cement them in my own head haha)